dependencies.gradle

```diff
@@ -4,6 +4,7 @@ dependencies {
   implementation 'org.msgpack:msgpack-core:0.8.11'
   implementation 'org.java-websocket:Java-WebSocket:1.4.0'
   implementation 'com.google.code.gson:gson:2.5'
+	implementation 'com.davidehrmann.vcdiff:vcdiff-core:0.1.1'
```

Comment:
is this a whitespace problem?

Reply:
Probably, yes. I'll look.

Reply:
@paddybyers so I just had to wrestle with IntelliJ to use two spaces here rather than the tab character that had been incorrectly inserted. But, to be fair to both IntelliJ and the original committer, the root problem lies [here](https://github.com/ably/ably-java/blob/4ef4ee64251c759c66c1456b37302ca73b5aa251/.editorconfig#L9).

FWIW, I feel strongly that *all* our codebases should consistently use spaces everywhere, never tabs. I seem to recall you may disagree, perhaps only for Java, I can't recall.

Reply:
Anyway, fixed: https://github.com/ably/ably-java/pull/550/commits/f057e6ca8011e160408770032a0a6cd8650f4ee3

Also, I've got this repo automatically dismissing approvals if new commits get added so can you please re-approve, @paddybyers  - sorry.